src/class_sqlThread.py

```diff
@@ -26,117 +26,134 @@
     from .debug import logger
     from .tr import _translate
 
+# pylint: disable=attribute-defined-outside-init,protected-access
+root_path = os.path.dirname(os.path.dirname(__file__))
 
-class sqlThread(threading.Thread):
-    """A thread for all SQL operations"""
-
-    def __init__(self):
-        threading.Thread.__init__(self, name="SQL")
-
-    def run(self):  # pylint: disable=too-many-locals, too-many-branches, too-many-statements
-        """Process SQL queries from `.helper_sql.sqlSubmitQueue`"""
-        helper_sql.sql_available = True
-        self.conn = sqlite3.connect(state.appdata + 'messages.dat')
-        self.conn.text_factory = str
-        self.cur = self.conn.cursor()
-
-        self.cur.execute('PRAGMA secure_delete = true')
 
-        # call create_function for encode address
-        self.create_function()
+class UpgradeDB(object):
+    """
+        Upgrade Db with respect to versions
+    """
 
+    def __init__(self):
+        self._current_level = None
+        self.max_level = 11
+        self.conn, self.cur = self.connection_build('messages.dat')
+
+    def connection_build(self, file_name=None):
```

Comment:
```
def connection_build(self, file_name='messages.dat'):
```

Reply:
applied changes method name to ```__connection_build_internal``` and pass file_name